security/models.py

```diff
@@ -27,13 +28,17 @@ def create_superuser(self, email, password, **extra_fields):
 
 
 # The fields (password, is_active, username) are included in AbstractBaseUser
-class User(AbstractBaseUser):
+class User(AbstractBaseUser, PermissionsMixin):
     email = models.EmailField(
         unique=True
     )
     objects = UserManager()
+    name = models.CharField(max_length=100, default='fname lname')
     completedTutorial = models.BooleanField(default=False)
     receivesAlerts = models.BooleanField(default=False)  # Todo, would this be better as its own table relation?
+    deleteFlag = models.BooleanField(default=False)
+    is_active = models.BooleanField('active', default=True)
```

Comment:
What is `is_active` being used for? Sounds like it's the same as the deleteFlag

Reply:
It seems to me that if you want to use the Django User default class, the user must have an is_active attribute. Maybe I'm wrong.

Reply:
It's already a trait that is inherited, you don't need to add it. View the commented line above

Reply:
Remove is_active